### PR TITLE
Get basic information for a page, shorthand for getting page likes.

### PR DESCRIPTION
**Pull request by asolove**
A small feature I needed for a project I'm working on.

I'm brand new to Elixir, so happy to receive any feedback on code style, etc.

### CODE REVIEW

File: lib/facebook.ex [hunk 1]
```diff
@@ -114,6 +114,43 @@ defmodule Facebook do
...
+
+	@spec page(page_id :: integer | String.t, access_token) :: response
+	def page(page_id, access_token) do
+		page(page_id, access_token, [], [])
+	end
+
+	@spec page(page_id :: integer | String.t, access_token, fields, options) :: response
+	def page(page_id, access_token, fields, options) do
+		params = [fields: fields]
+		if !is_nil(access_token) do
```
Comment (mweibel) [inline lib/facebook.ex]: do you need that check? can you fetch the page without having an access_token?

File: lib/facebook.ex [hunk 2]
```diff
@@ -114,6 +114,43 @@ defmodule Facebook do
...
+	end
+
+	@doc """
+	Basic page information for the provided page_id
+
+	See: https://developers.facebook.com/docs/graph-api/reference/page
+	"""
+	@spec page(page_id :: integer | String.t) :: response
+	def page(page_id) do
+		page(page_id, nil, [], [])
```
Comment (mweibel) [inline lib/facebook.ex]: probably here you should also accept an access_token, but if the access_token is not required (see other comment), we can leave it

**Comment by asolove** [comment]
You're exactly right so I've removed the unnecessary function definition and nil check. Also added a few smaller items that are not directly related to the PR but would be nice.

**Comment by mweibel** [comment]
Neat :) thanks. Published version 0.3.0

**Pull request merged**